### DISCUSSION TITLE
Create Sections with Templates

### DISCUSSION THREAD

**Discussion by trendformer** [Laravel-Backpack/community-forum#254] [Q&A (Help)]
Hi!

I'm looking for a way to implement sections with the repeater field (or something like that)...

We must be able to select a "section" with defined subfields.
The sections should then be displayed one below the other in the frontend.

I hope you understand what I mean...
Anybody 

Thanks!

**Comment by jcastroa87** [comment]
Hello @trendformer if you still need this, can you draw a mockup to understand the goal?

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
